pnpm-workspace.yaml

```diff
@@ -2,4 +2,5 @@ packages:
   - "packages/config/*"
   - "packages/*"
   - "examples/*"
+  - "examples/express/client"
```

Comment:
I think changing L#4 to `"examples/**` would be the generic way to accomplish this?

Reply:
it's kinda hard to have nested pnpm workspaces actually, in trpc we use npm workspaces for the nested examples:

https://github.com/trpc/trpc/blob/64f28b6b612889f7785cbab4cab9d8cc1df70c99/examples/minimal-react/package.json#L5-L8